tests/slack_notifier/utils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import pycurl
+
+class SlackTestHandler(logging.StreamHandler):
+    
+    '''Custom handler inheriting from StreamHandler to retain the
+    formatter functionality.
+    Overriding the emit function to use pycurl to send messages to slack
+    (as it turned out to be the most robust option).
+    '''
+    c = pycurl.Curl()
```

Comment:
Decided to leave the SlackTestHandler as similar to the real SlackHandler as possible (i.e. only overwriting the emit() method). I can't see how leaving the pycurl.Curl() object in the handler could make it crash, but I feel like it's better to have tests to confirm that it does not happen.